cds_ils/migrator/relations/api.py

```diff
@@ -41,6 +44,32 @@ def create_parent_child_relation(parent, child, relation_type, volume):
     )
 
 
+def check_for_special_series(record):
+    """Link documents and serials (DESIGN REPORT/YELLOW REPORT)."""
+    series_class = current_app_ils.series_record_cls
+    design_report = {"title": "DESIGN REPORT",
+                     "volume": None,
+                     "issn": None,
+                     }
+    yellow_report = {"title": "YELLOW REPORT",
+                     "volume": None,
```

Comment:
are you sure it will always be volume `None`? were there any cases in the data that have assign volume and/or issn?

Reply:
When we translate it, we alwasy set it to None, both volume and issn, [check here](https://github.com/CERNDocumentServer/cds-ils/pull/272/files#diff-e1bf25cf864e904c8fcd2e30967b9c566fceef148caf08d99d6720d66935da04R138),  I don't know the marcxml field that would correspond to volume or issn, in the [issue](https://github.com/CERNDocumentServer/cds-dojson/issues/209) it's only mentioned to convert them to series (unless I am missing something)

Reply:
I checked with the library, we don't have the volume in this case, but it was better to make sure.